ocs_ci/ocs/cluster_load.py

```diff
@@ -327,11 +327,20 @@ def get_query(self, query, mute_logs=False):
         """
         now = datetime.now
         timestamp = datetime.timestamp
-        return float(
-            self.prometheus_api.query(
-                query, str(timestamp(now())), mute_logs=mute_logs
-            )[0]['value'][1]
-        )
+
+        counter = 0
+        while counter < 5:
```

Comment:
what about to just use re-try here in whole function? We can add also more exceptions to make it more robust if needed

Reply:
done